src/lib/constants.ts

```diff
@@ -119,6 +119,7 @@ export const enum Emoji {
 	Ranged = '<:ranged:630911040258834473>',
 	Gear = '<:gear:835314891950129202>',
 	Slayer = '<:slayer:630911040560824330>',
+	Clue = '<:Clue_scroll:365003979840552960>',
```

Comment:
```suggestion
	Clue = '<:clue_scroll:365003979840552960>',
```
small letter for consistency.